toolkit/resources/manifests/package/toolchain_x86_64.txt

```diff
@@ -518,12 +516,6 @@ procps-ng-3.3.15-5.cm1.x86_64.rpm
 procps-ng-debuginfo-3.3.15-5.cm1.x86_64.rpm
 procps-ng-devel-3.3.15-5.cm1.x86_64.rpm
 procps-ng-lang-3.3.15-5.cm1.x86_64.rpm
-python2-2.7.18-4.cm1.x86_64.rpm
-python2-debuginfo-2.7.18-4.cm1.x86_64.rpm
-python2-devel-2.7.18-4.cm1.x86_64.rpm
-python2-libs-2.7.18-4.cm1.x86_64.rpm
-python2-test-2.7.18-4.cm1.x86_64.rpm
```

Comment:
You'll also need to update `toolkit/resources/manifests/package/update_manifests.sh`.